### PR TITLE
Update README with more typical startup steps

### PR DESCRIPTION
**Pull request by botimer**

### CODE REVIEW

**Comment by daaang** [comment]
ok this got all weird, anyway basically just prepend your commit message with `:memo:` and it'll be consistent with my usual emoji formatting

**Comment by botimer** [comment]
Cool. I updated the message with 📝 and rebased/pushed the branch. I think it's good to go.

File: README.md [hunk 1]
```diff
@@ -20,9 +20,8 @@ Tutorial for a new project
 --------------------------
 
 ```bash session
-$ mkdir path/to/your/new/project
-$ cd path/to/your/new/project
-$ git clone git@github.com:mlibrary/node-quickstart .
+$ git clone https://github.com/mlibrary/node-quickstart your-project
+$ cd your-project
```
Comment (daaang) [inline README.md:24]: this is good, it's a much better version of what i wrote, yes

**Review by daaang — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (botimer) [reply README.md:24]: Oh wow! He Likes It! 👍

**Pull request merged**